build/charts/antrea/values.yaml

```diff
@@ -89,6 +89,12 @@ nodeIPAM:
   # -- Mask size for IPv6 Node CIDR in IPv6 or dual-stack cluster.
   nodeCIDRMaskSizeIPv6: 64
 
+ipsecCSRSigner:
+  # -- Auto approve policy of Antrea signer
+  autoApprovePolicy: "validateAgentPodName"
+  enable: false
+  selfSignedCA: true
```

Comment:
there is already an `ipsec` config section above. Can we keep related things together to avoid adding too many config sections? The structure of this valuies.yaml file doesn't need to match the structure of the yaml config stored in the ConfigMap.

Reply:
That existing "ipsec" section is for agent, but this one is for controller. Are we able to share?

Reply:
I think it should be based on the feature and not which Antrea component is in charge of the implementation. See other sections for `nodeIPAM`, `nodePortLocal`, etc.

Also based, on the current configuration parameters, it is not clear to me how one can choose between psk mode and certificate mode.

Reply:
My suggestion is if psk is provided, then PSK mode; otherwise certificate mode. What you think?

Reply:
That works for me too.

Reply:
thanks for the suggestion!